Gemfile

```diff
@@ -108,6 +108,7 @@ group :development do
 end
 
 group :test do
+  gem "database_cleaner"
```

Comment:
Cypress tests can leave behind quite a few records in the database, ruining all further rspec tests.